doc/nrf-bm/doxygen/nrf-bm.doxyfile

```diff
@@ -2402,6 +2402,7 @@ INCLUDE_FILE_PATTERNS  =
 
 PREDEFINED             = __DOXYGEN__ \
                          CONFIG_BLE_QWR_MAX_ATTR=1 \
+                         CONFIG_BLE_SCAN_FILTER=1 \
```

Comment:
why? We don't do this in zephyr and we don't do this in NCS, so why is this required in BM?